CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 6.15.0 - 2024-10-11
+### Added
+- Add ReportDefinitionAqtsLicense to ReportDefinition
```

Comment:
My only complaint would be that this was actually added to EnumReportDefinitionCategory, ReportDefinition is something different